apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -125,12 +127,21 @@ var TopInstructions = React.createClass({
    * @returns {number}
    */
   adjustMaxNeededHeight() {
-    const instructionsContent = this.refs.instructions;
-    const maxNeededHeight = $(ReactDOM.findDOMNode(instructionsContent)).outerHeight(true) +
-      HEADER_HEIGHT + RESIZER_HEIGHT;
+    if (this.props.hasContainedLevels) {
+      const maxNeededContainedLevelHeight =
+        $(ReactDOM.findDOMNode(this)).find('#containedLevelContainer').outerHeight(true) +
+        HEADER_HEIGHT + RESIZER_HEIGHT;
```

Comment:
Would prefer to give containedLevelContainer a ref, and then do `$(ReactDOM.findDOMNode(this.refs.containedLevel).outerHeight(true)`